components/core/src/clp/ffi/ir_stream/search/test/utils.cpp

```
@@ -0,0 +1,161 @@
+        SchemaTree::Node::Type type
+) const -> std::vector<SchemaTree::Node::id_t> {
+    std::vector<SchemaTree::Node::id_t> node_ids;
+    for (auto const node_id : m_matchable_node_ids) {
+        if (m_schema_tree->get_node(node_id).get_type() == type) {
+            node_ids.emplace_back(node_id);
+        }
+    }
+    return node_ids;
+}
+
+auto ColumnQueryPossibleMatches::get_projectable_node_ids() const
```

Cppcheck (2.10-2)

[style] The function 'get_projectable_node_ids' is never used.

(unusedFunction)